client/src/index.js

```diff
@@ -15,6 +15,8 @@ socket.addEventListener('message', function (event) {
 
 function appendTextDiv(parent,text){
   var div = document.createElement('div');
+  div.setAttribute("class","list-group-item list-group-item-primary");
```

Comment:
Let's refactor this a bit and use innerHTML for help. This way we will see in code exactly what the browser will contain. This is called templating. Note that I am not using forEach anymore, but .map() to go through each element of the array, then map it into another element and return the array. Then we call join to get a string.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join

```
mainDiv.innerHTML = obj.answers.map((answers) => getAnswerHTML(answers.answer)).join('');
```

This function will return a String! not a HTML, as the innerHTML accepts strings, the browser turns them into HTML.
```
function getAnswerHTML(text) {
    return `<div  style="fit-content" class="list-group-item list-group-item-primary">${text}</div>`
}
```

Reply:
Also https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
So, You mean under refactor that we will not use appendTextDiv function anymore? I have finished with new getAnswerHTLM function and it works. Should I have to remove appendTextDiv function?